### PR TITLE
Add simple click event using the “on” method

### PR DESCRIPTION
**Pull request by gabycampagna**
When user clicks on the donut chart it will enlarge. The panel container isn't enlarging yet...We need to discuss this further, but just wanted to get us started on this.

### CODE REVIEW

**Comment by gabycampagna** [comment]
Added "path.on" and removed "chart.on"

File: addon/styles/_variables.scss [hunk 1]
```diff
@@ -1,9 +1,24 @@
 //default properties
+$brown: #a52a2a;
```
Comment (jrjohnson) [inline addon/styles/_variables.scss]: I'm not sure why these variables are included here. Are they needed for something? I don't see that in the rest of the code.

File: tests/dummy/app/router.js [hunk 2]
```diff
@@ -7,6 +7,14 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
+  this.route('simple-chart-donut');
```
Comment (jrjohnson) [inline tests/dummy/app/router.js]: I think the routes are a little bit more understandable if they are just `donut` and not `simple-chart-donut`.

File: addon/styles/_variables.scss (hunk 1 above)
Comment (gabycampagna) [inline addon/styles/_variables.scss]: This one is for the hover on the .bars class.

File: addon/styles/components/simple-chart-bar.scss [hunk 3]
```diff
@@ -1,4 +1,4 @@
-.simple-chart-bar {
```
Comment (gabycampagna) [inline addon/styles/components/simple-chart-bar.scss]: I moved the .bar class to the  ```simple-chart.scss``` Other charts will be using it.

File: tests/dummy/app/styles/app.scss [hunk 4]
```diff
@@ -48,6 +48,14 @@ body {
   }
 }
 
+.large-panel {
```
Comment (gabycampagna) [inline tests/dummy/app/styles/app.scss]: Oh, yes!

Comment (gabycampagna) [inline tests/dummy/app/styles/app.scss]: Removed.

File: tests/dummy/app/router.js [hunk 5]
```diff
@@ -7,6 +7,13 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
+  this.route('donut');
```
Comment (gabycampagna) [inline tests/dummy/app/router.js]: Should I remove the new handlebar files too and then add a new route and handlbar file for only the donut-chart in the new PR?

Comment (gabycampagna) [inline tests/dummy/app/router.js]: I will removed the generated router/hbs files and open a new PR for them.

File: addon/components/simple-chart.js [hunk 6]
```diff
@@ -34,6 +34,14 @@ export default Component.extend({
         this.set('tooltipLocation', tooltipLocation);
       }
     },
+    click(data, slice, tooltipLocation){
```
Comment (gabycampagna) [inline addon/components/simple-chart.js]: I did work with one on another app in the past. (see screenshot)
screen shot 2017-06-29 at 4 44 29 pm
 I can remove it and you can always add it back later if you ever find a need for one.

Comment (jrjohnson) [inline addon/components/simple-chart.js]: Nope. That is enough for me, lets include it! I just have no imagination.

Comment (gabycampagna) [inline addon/components/simple-chart.js]: Awesome!  Thanks for merging!

**Pull request merged**